src/lib/webworker.generated.d.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference lib="webworker.importscripts" />
```

Comment:
this is not needed here, the file is autogenerated, and it already has the correct definition. we add it to lib.d.ts to partially allow the lib to be used in webworkers for simple cases.. 

the two files `dom` and `webworker` have too many conflicts in between them, and can not be loaded together in the same context anyways..

so I would just revert the changes to this file, and leave it as is.